views/forms/MultiSelect/index.jsx

```diff
@@ -0,0 +1,47 @@
+import { Autocomplete, Chip, TextField } from '@mui/material';
+import React from 'react';
+
+export const MultiSelect = (props) => {
+  const { field, fieldState } = props;
+  
+  return (
+      <Autocomplete
+        id={props.name}
+        fullWidth={props.fullWidth}
+        getOptionDisabled={(option) => option.isFixed}
+        isOptionEqualToValue={(option, value) => option.value === value.value}
+        multiple
+        onChange={(_, data, reason) => props.onChange(data, reason, field)}
+        options={props.options}
+        renderTags={(tagValue, getTagProps) =>
+          tagValue.map((option, index) => (
+            <Chip
+              key={`${field}-chip-${index}`}
+              label={option.label}
+              {...getTagProps({ index })}
+              disabled={option.isFixed}
+            />
+          ))
```

Comment:
It should render the chips without this logic, it's mostly checking wether a value is persistent or not

Reply:
This is straight pulled from the existing ControlledMultiSelect component, I can't remove it without causing a regression there.